src/main/java/Duke.java

```diff
@@ -1,10 +1,200 @@
+import java.util.Scanner;
+
 public class Duke {
+
+    /** Displays messages */
+    private static final String BORDER = "____________________________________________________________";
+    private static final String NEWLINE = System.lineSeparator();
+    private static final String DONE_STATUS = "X";
+
+    public static void greet() {
+        System.out.println(BORDER);
+        help();
+        System.out.println("Hello, I'm Panda!");
+        System.out.println("What would you like to do today?");
+        System.out.println(BORDER + NEWLINE);
```

Comment:
For each function there is repeat `System.out.println(BORDER);`
Maybe this can be moved to before Duke.c#L189 and after Duke.c#L196, since anyway these two will be printed each time an input is received?